examples/src/clipping.ts

```diff
@@ -23,10 +23,10 @@ async function main() {
   document.body.appendChild(renderer.domElement);
 
   let modelsNeedUpdate = true;
-  const revealManager = new RevealManager(client, () => {
+  const revealManager = new reveal.RevealManager(client, () => {
```

Comment:
Should we stick with RevealManager, or should we go for ModelManager, or Model3DManager or something along those lines?

Reply:
Not part of this PR, but we should maybe discuss the name at some time. I don't like "Manager" myself.